numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -72,6 +72,41 @@ def check_for_different_datatypes(
     return True
 
 
+def check_for_different_datatypes_array_creations(
```

Comment:
A separate function has been created, since the testing mechanics are different from the basic one.